docs/index.asciidoc

```diff
@@ -4,20 +4,21 @@
 //////////
 release-state can be: released | prerelease | unreleased
 //////////
-:release-state:  prerelease
-:version:        6.0.0-alpha1
-:major-version:  6.x
-:branch:         master
-
-:docker-image:   docker.elastic.co/kibana/kibana:{version}
-:es-ref:         https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
-:xpack-ref:      https://www.elastic.co/guide/en/x-pack/current/
-:repo:           https://github.com/elastic/kibana/
-:issue:          {repo}issues/
-:pull:           {repo}pull/
-:commit:         {repo}commit/
-:blob:           {repo}blob/{branch}/
-:security:       https://www.elastic.co/community/security/
+:release-state:      prerelease
+:version:            6.0.0-alpha1
+:major-version:      6.x
+:branch:             master
+
+:docker-image:       docker.elastic.co/kibana/kibana:{version}
+:es-ref:             https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
+:xpack-ref:          https://www.elastic.co/guide/en/x-pack/current/
+:ui-framework-ref:   https://www.elastic.co/kibana/{version}/ui-framework
```

Comment:
Our other docs all publish based on branch rather than exact version number.  Is there some reason why the ui framework should be handled differently?